themes/default/content/blog/2022-03-30-introducing-pulumiverse/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Introducing the Pulumiverse"
+date: 2022-03-30T10:00:00Z
+draft: false
+meta_desc: The Pulumiverse is a community organized project to help people contribute and use Pulumi more effectively
+meta_image: meta.png
+authors: ["david-flanagan"]
+tags: ["Community"]
+---
+
+# Introducing the Pulumiverse
+
+Today, we're excited to announce that we're working with the Pulumi community to provide a place to interact and collaborate on Pulumi-based libraries, projects, and educational materials: the [Pulumiverse](https://github.com/pulumiverse).
+
```

Comment:
```suggestion

<!--more-->

```

Reply:
What does this do?

Reply:
It tells Hugo how much of the article should show up in the preview of the article on the main blog page.